os/fs/driver/mtd/smart.c

```diff
@@ -4040,7 +4072,7 @@ static int smart_validate_crc(FAR struct smart_struct_s *dev)
 	}
 #else
 	/* Test 8-bit CRC for CRC8 & basic case for smart_scan. */
-	if (crc != header->crc8) {
+	if (crc != header->crc8 && crc != SMART_ERASEDSTATE_CRC) {
```

Comment:
This line means that both header->crc8 and SMART_ERASESTATE_CRC should be equal to crc (newly calculated one).
Then when the case 'header->crc8 != SMART_ERASEDSTATE_CTC' can happen?
(if the target sector is corrupted due to erase failure or power reset.,?)

Reply:
Hi Chanhee,
I'll drop a mail to you to clarify regarding this.